modules/nodes-containers-volumes-listing.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-containers-volumes.adoc
+
+[id='nodes-containers-volumes-listing_{context}']
```

Comment:
This feels less like a procedure topic and more like a CLI reference topic. You have several of these. They don't really fit well into flows, but they're great reference material.